examples/provider.yaml

```diff
@@ -0,0 +1,6 @@
+apiVersion: pkg.crossplane.io/v1
+kind: Provider
+metadata:
+  name: provider-aws
+spec:
+  package: crossplane/provider-aws:alpha
```

Comment:
Does `alpha` work?

Reply:
@muvaf yes, we promote provider releases to `alpha` channel. Example [here](https://hub.docker.com/layers/crossplane/provider-aws/alpha/images/sha256-40325541e89db29a60605c40f1de36af5c7fad02702fda51e7be7c525ef60dd9?context=repo) (matches digest of `v0.16.0`).